civilcode.shrc

```diff
@@ -42,9 +44,22 @@ else
 		app="$1"; shift
 		_docker_compose_exec -e MIX_ENV=test application bash -c "cd /app/apps/$app && $*"
 	}
-
 fi
 
+function _prepare_slack_checkout() {
+	# Optional argument: date (format YYYYMMDD)
+	date="${1:-$(date +"%Y%m%d")}"
+	github_user=$(security find-internet-password -s github.com | grep acct | sed 's/^.*="\(.*\)".*$/\1/')
+
+	base_hub_options=(--include-pulls -a $github_user -d $date)
+	open_issues=$(hub issue $base_hub_options -s open -f "• in-progress: _%t_; add details%n%U%n")
+	closed_issues=$(hub issue $base_hub_options -s closed -f "• completed: _%t_; add details%n%U%n")
```

Comment:
Shall we change the `add details` to `ADD_DETAILS` so it is easier to spot and can be replaced with one?

Reply:
I would suggest leaving 'add details' out...  I think that's self-explanatory

Reply:
I'm in favour of leaving it in. Nothing wrong with having prompts.

Reply:
Like @bgracie I'm in favor of removing the add details. 
I think most of the time, we don't necessarily add details. If necessary we know how to add those on slack.
So for most of the time, having no prompts = less effort.

Reply:
We should be providing a bit more details in our check-outs, especially for the ones "in-review" or "in-progress", just to let the others know a quick status on the issue. i.e. `ready for review by @...` or `waiting on Giverny to reply` or `almost done, need to do a final code review`. It is hard for me in the morning to know what is going on with the on-going checkout messages from the previous day, and when I update the tracking tool.

Reply:
Ok.. Trade-off: STATUS placeholder at the end only for open issues.